scripts/develop.sh

```diff
@@ -3,9 +3,9 @@
 cd "$(dirname "${BASH_SOURCE[0]}")"
 set -euxo pipefail
 
-./build.sh
+./build.sh || exit 1
 
 docker run --rm -it \
-    -e GIT_URL=https://github.com/denoland/deno \
-    -e INIT_SCRIPT="bash" \
+    -e ENVBUILDER_GIT_URL=https://github.com/denoland/deno \
```

Comment:
Is it a breaking change?

BTW We may need to update README file too.

Reply:
This would only be a breaking change if you were using the `develop.sh` script with a several-month-old old version of envbuilder. I don't think this counts.

Reply:
README was updated in https://github.com/coder/envbuilder/pull/203